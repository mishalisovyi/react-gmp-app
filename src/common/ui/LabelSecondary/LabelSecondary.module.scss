@use '../../../mixins';
@use '../../../colors';

.LabelSecondary {
  @include mixins.typography-subtitle-small;

  padding: 4px 8px;
  color: colors.$white;
  opacity: .7;
  border: solid 1px colors.$stone-cold;
  border-radius: 4px;
}