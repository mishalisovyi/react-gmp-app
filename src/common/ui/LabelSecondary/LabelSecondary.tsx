import React, { memo } from 'react';
import styles from './LabelSecondary.module.scss';

interface LabelSecondaryProps {
  value: string | number;
}

function LabelSecondaryComponent({ value }: LabelSecondaryProps) {
  return <span className={styles['LabelSecondary']}>{value}</span>;
}

export const LabelSecondary = memo(LabelSecondaryComponent);
