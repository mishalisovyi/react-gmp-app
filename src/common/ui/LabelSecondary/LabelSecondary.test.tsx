import React from 'react';
import renderer from 'react-test-renderer';

import { LabelSecondary } from './LabelSecondary';

it('should render secondary label', () => {
  const tree = renderer.create(<LabelSecondary value="test" />);

  expect(tree.toJSON()).toMatchSnapshot();
});
