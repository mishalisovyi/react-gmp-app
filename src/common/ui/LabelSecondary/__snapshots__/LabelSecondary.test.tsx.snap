// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render secondary label 1`] = `
<span>
  test
</span>
`;
